src/CattyTests/Setup/Firebase/FirebaseCrashlyticsSetupTests.swift

```diff
@@ -145,4 +145,15 @@ final class FirebaseCrashlyticsSetupTests: XCTestCase {
         XCTAssertEqual(0, crashlytics!.logs.count)
         XCTAssertEqual(1, crashlytics!.records.count)
     }
+
+    func testProjectSearchFailureNotification() {
+        UserDefaults.standard.set(true, forKey: kFirebaseSendCrashReports)
+        app?.setupCrashlytics()
+
+        NotificationCenter.default.post(name: .projectSearchFailure, object: nil)
+
+        XCTAssertEqual(0, crashlytics!.logs.count)
```

Comment:
Please verify the `userInfo` content to call attention to the error like mentioned above (e.g. like in `testProjectFetchFailureNotification`).